co-views/co-view.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for co-views v2.1
+// Project: https://github.com/tj/co-views/
+// Definitions by: devlee <https://github.com/devlee/>
+// Definitions: https://github.com/devlee/DefinitelyTyped
+
+/* =================== USAGE ===================
+
+    import views = require('co-views');
+    const render = views('views', {
+        map: {
+            html: 'swig'
+        },
+        default: 'jade'
+    });
+
+ =============================================== */
+
+declare module "co-views" {
+
+    interface IEngineMap {
```

Comment:
https://www.typescriptlang.org/docs/handbook/writing-declaration-files.html#naming-conventions
> Naming Conventions
In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.